packages/core/src/Box/Box.spec.tsx

```diff
@@ -49,4 +49,10 @@ describe('Box', () => {
     expect(json).toMatchSnapshot()
     expect(json).toHaveStyleRule('box-shadow', theme.shadows.sm)
   })
+
+  test('overflow prop sets overflow', () => {
```

Comment:
n-b but if possible, create snapshots with `asFragment` from testing-library. we want to eventually remove `rendererCreateWithTheme`